### DISCUSSION TITLE
Suddenly lost access to Github Copilot student offer

### DISCUSSION THREAD

**Discussion by erbanku** [community/community#47637] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I suddenly lost access to the Github Copilot student offer, which I had been previously using. I'm not sure why this happened, but I was wondering if there's anything I can do to regain access to the offer.

I appreciate any assistance you can provide with this matter
> Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended.
![image](https://user-images.githubusercontent.com/79493862/219337619-17c2ae24-79f8-467c-8a4c-6786b285159f.png)

**Comment by ivan-plotnikov** [comment]


**Reply by erbanku** [reply]
Resolved itself.
